t/jv-boolean.t

```diff
@@ -0,0 +1,31 @@
+use lib '.';
+use t::Helper;
+use Test::More;
+use utf8;
+use JSON;
+
+my $schema = {
+  type       => 'object',
+  properties => {nick => {type => 'boolean'}}
+};
+
+validate_ok {nick => JSON::true}, $schema;
+validate_ok {nick => 1000},       $schema, E('/nick', 'Expected boolean - got number.');
+validate_ok {nick => 0.5},        $schema, E('/nick', 'Expected boolean - got number.');
+validate_ok {nick => 'nick'},     $schema, E('/nick', 'Expected boolean - got string.');
+validate_ok {nick => bless({}, 'BoolTest')},   $schema;
+validate_ok {nick => bless({}, 'BoolTestNegative')},   $schema, E('/nick', 'Expected boolean - got BoolTestNegative.');
```

Comment:
What's the reason for testing both "BoolTest" and "BoolTestNegative". What's the difference between the two tests?